README.md

```diff
@@ -314,7 +314,7 @@ Remember: Without strong encryption, you will be spied on systematically by lots
 **Check out [this site](https://www.securemessagingapps.com/) for comparisons*.
 
 <img width="16" src="misc/forbidden.png"> </img> **Avoid**
-- Whatsapp
+- WhatAapp
```

Comment:
Thanks for the fixes, could you correct this typo here before I merge this?

Reply:
Done, thank you!